wifi-menu: add page (#6202)
tldr-pages/tldr: pages/linux/wifi-menu.md

# wifi-menu

> > An easy to use interactive menu to setup network.

Reviewer: I think this is more descriptive:
```suggestion
> Interactively connect to a wireless network.
```

> More information: <https://manned.org/wifi-menu>.

> - Setup a wireless connection interactively:

Reviewer 2: ```suggestion
- Set up a wireless connection interactively:
```

> - Interactively setup a connection to a network:

Reviewer: Perhaps?
```suggestion
- Setup a wireless connection interactively:
```

`wifi-menu`

> - Interactively setup a connect to a network and obscure the password:

Reviewer 2: ```suggestion
- Interactively set up a connection to a network and obscure the password:
```
```suggestion
- Interactively connect to a network and obscure the password:
```
Either are fine

> `wifi-menu --obscure`

Reviewer 3: Wait why is this an option?! It should be enabled by default.

Author: Because who doesn't want someone peering over their shoulder at their network password
